docs/navbar.html

```diff
@@ -1,7 +1,7 @@
 <div class="navbar">
-  <div id="nav-left" class="nav-left"></div>
   <div class="nav-title"><h1 class="nav-header" id="nav-title-header"></h1></div>
-
+  <div id="nav-left" class="nav-left"></div>
```

Comment:
Is this what we want? I feel like the disaster picker should be less prominent in general, since it's usually the same for weeks/months, but this feels like it's more prominent. Can you say what you want to accomplish with this change?